BPM+/SCE-Semantic.xsd

```diff
@@ -852,11 +840,30 @@
 			</xsd:extension>
 		</xsd:complexContent>
 	</xsd:complexType>
+
+	<xsd:element name="RelationshipKindSet" type="sce:tRelationshipKindSet"/>	
+	<xsd:complexType name="tRelationshipKindSet">
+		<xsd:complexContent>
+			<xsd:extension base="sce:tSCEKindSet">
+				<xsd:sequence>
+					<xsd:element ref="sce:RelationshipKind" minOccurs="0" maxOccurs="unbounded">
+						<xsd:annotation>
+							<xsd:documentation>
+								Element Role: kind
+								The extendable list of enumerated terms is a set of Kinds.
+								This redefines the kind association of SCEKindSet.
+							</xsd:documentation>
+						</xsd:annotation>
+					</xsd:element>
+				</xsd:sequence>				
+			</xsd:extension>
+		</xsd:complexContent>
+	</xsd:complexType>
```

Comment:
This belongs to SCE-25

Reply:
The [proposal SCE-70 is to not change RelationshipKindSet](https://issues.omg.org/browse/SCE-70)